src/JsonSchema/Constraints/Constraint.php

```diff
@@ -24,12 +24,12 @@
 {
     protected $inlineSchemaProperty = '$schema';
 
-    const CHECK_MODE_NONE =             0x00000000;
-    const CHECK_MODE_NORMAL =           0x00000001;
-    const CHECK_MODE_TYPE_CAST =        0x00000002;
-    const CHECK_MODE_COERCE_TYPES =     0x00000004;
-    const CHECK_MODE_APPLY_DEFAULTS =   0x00000008;
-    const CHECK_MODE_EXCEPTIONS =       0x00000010;
+    const CHECK_MODE_NONE = 0x00000000;
```

Comment:
@alcohol are there comment delimiters or other facility to skip code blocks for php-cs-fixer?